Update to slnx

stefannikolei opened the pull request:
This upgrades all the solution files to the new Solution Format .slnx 
In this PR the old files are not deleted. 

This should be deleted in the near future. Otherwise they would run out of sync.

The Dependencies look good in my eyes. Omg is it easy to read now xD

Fixes #704

stefannikolei committed:
Update to slnx

This upgrades all the solution files to the new Solution Format .slnx
In this PR the old files are not deleted. This should be deleted in the near future. Otherwise they would run out of sync.

stefannikolei committed:
Use .slnx in the builds

stefannikolei committed:
Merge branch 'master' into sn/slnx

twostars reviewed (changes requested):
I can't really see much else other than that, for what exists.
What I do wonder, though, is can we pull in consistent project blocks into a solution, so we don't have to re-define their dependencies everywhere?

A lot of the hassle here is ensuring the dependency chains are setup consistently. I should imagine it should be able to import, though how the IDE would like that I don't know.

stefannikolei committed:
Update readme and codeql

stefannikolei committed:
Remove sln's update llfio

stefannikolei committed:
Address PR feedback

Removed FileIO as BuildDependency for KscViewer
Added dependencie sync-submodule to fetch-argparse.vcxproj

twostars committed:
Let VS attach the project IDs
